src/cascadia/TerminalApp/FilteredCommand.idl

```diff
@@ -6,7 +6,7 @@ import "HighlightedTextControl.idl";
 
 namespace TerminalApp
 {
-    [default_interface] runtimeclass FilteredCommand : Windows.UI.Xaml.Data.INotifyPropertyChanged
+    [default_interface] unsealed runtimeclass FilteredCommand : Windows.UI.Xaml.Data.INotifyPropertyChanged
```

Comment:
Oh, it's not about inheriting from `INotifyPropertyChanged`. It's so that `FilteredTask` can inherit from `FilteredCommand`. Because a `FilteredCommand` does almost everything we want in this pane, but I wanted to override just a few pieces of it.

Reply:
Would it make sense to do something like this?
```cpp
runtimeclass FilteredTask {
    FilteredCommand Command;
    // ... remaining members ...
}
```

I checked the diff, and it didn't seem like we're upcasting a `task` into a `command` anywhere, nor `try_as`ing a `command` into a `task`. So, do we actually need inheritance specifically or would composition be sufficient as well?

Reply:
done in 5ec555af5, cause yea that was janky